### ISSUE TITLE
Order translators by language name instead of translator name

### ISSUE THREAD

**Issue by scootergrisen** [lxde/lxqt#1456]
##### Expected Behavior
I would find et better to arrange the translator names by languages.
```
Danish
    Some name
    Some other navn
Norwegian
    Some name
    Some other navn
```

##### Steps to Reproduce (for bugs)
1. open program menu
2. select About LXQt
3. select translators tab

##### System Information

* Distribution & Version: siduction-18.2.0-patience-lxqt-amd64-201803072340
* Qt Version: 0.12.0

**Comment by tsujan** [comment]
I agree. When people are interested in that part, they want to know who translated in which language and it's safe to assume that the majority of LXQt users don't have ESP.

**Comment by agaida** [comment]
Full ACK - @scootergrisen feel free to create a PR

**Comment by scootergrisen** [comment]
Sadly i never learned C++.
